after/syntax/javascript.vim

```diff
@@ -21,24 +20,45 @@ if exists('s:current_syntax')
   let b:current_syntax = s:current_syntax
 endif
 
-" <tag key={this.props.key}>
-"          s~~~~~~~~~~~~~~e
-syntax region jsxEscapeJs
-      \ contained
-      \ contains=jsBlock
-      \ start=+{+
-      \ end=++
-      \ extend
+" check the javascript syntax file and cache it
+if !exists('s:js_syntax')
+  let syntax_list = execute('syntax list')
```

Comment:
why use global variable?
this line outside function, so should script local variable, actual: `s:syntax_list`.

Reply:
Changed.

Reply:
Thanks!